article/sections/_1_intro_generic.tex

```diff
@@ -0,0 +1,48 @@
+\section{Introduction}
+\label{sec:intro}
+
+In transportation, we often build models that are used for policy evaluation.
+Specifically, we often use behavioral models to evaluate the impact of external interventions on travel outcomes. 
+These evaluations are excplicitly causal: we are interested in how the 
+system reacts to \textit{interventions}. 
+Yet, when these models are 
+developed, causality is often not considered. Furthermore, when 
+causal concepts are accounted for, the process is done implicitly without a 
+formal framework. 
```

Comment:
Here, it seems like there is a disconnect from the previous sentences. You state "we" in the previous sentences, but create some distance here. Maybe the following?
```suggestion
Yet, when we develop these models, we often look causality.
Even in situations where we account for causality, we do so implicitly without using a formal framework.
```

Reply:
"Yet when we develop these models, we often **overlook** causality"